script.module.inputstreamhelper/default.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+''' This is the actual InputStream Helper API script entry point '''
+
+from __future__ import absolute_import, division, unicode_literals
+import os
+import sys
```

Comment:
Why is this needed?

Reply:
I mean the sys.path.append

Reply:
Because default.py (**xbmc.python.script** extension point) needs access to  to functions of the **xbmc.python.module** extension point` but is not in the same directory.
In 0.4.1 we imported script.module.inputstreamhelper into itself to solve this problem, but that caused the Kodi add-on information pane to freeze.

Reply:
Hum, you guys are likely doing something wrong then. There's no need to append to sys.path inside the same add-on. I'd recommend to take a deeper look in the future to see which imports caused the circular dependency in the first place. For now I'm fine with it.

Reply:
Or we have been bitten by an issue I have experienced myself various times before, which causes a just installed script.module to not be added to PYTHONPATH. We have seen this for different modules and add-ons, so I doubt it is an issue with one specific module or add-on.

Possibly related to Kodi using the repository metadata and not the package metadata (as reported by @mediaminister).

Reply:
@dagwieers I have also noticed this. Perhaps report it at https://github.com/xbmc/xbmc/issues/

Reply:
@basrieter Reported issues get closed without debug log, so I don't bother if it's not easily reproduced. Feel free to report it if you can.

Reply:
Well, you cannot expect devs to look into stuff without the proper information. Just provide them the info and people will look into it.

Reply:
It depends on how you get about with reported issues. But I am not expecting anything from volunteers, as a volunteer. But in this case I didn't bother. And you didn't either :wink: